### PR TITLE
personalized advice files

### PR DESCRIPTION
**Pull request by opal8000**
Adding my advice files

### CODE REVIEW

File: advice/saraht-advice.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!doctype html>
```
Comment (kerolloz) [inline advice/saraht-advice.html]: please write it as
<!DOCTYPE html>

**Comment by acodeinprogress** [comment]
Hey @opal8000,

This looks great! I just have 1 teensy request before I merge this in.
We would like for you to include a link to your advice page in the index.html file. If you could add that in, all you have to do is add, commit, and push your changes and they will join your pull request here. Once you do that, we can get it all merged in.

xoxo,
Sharon

**Comment by acodeinprogress** [comment]
Hey @opal8000,

Thanks for making that change. Unfortunately, it looks like you deleted a couple students' work in the index.html file. You can see the lines highlighted in pink by clicking the [Files Changed](https://github.com/skillcrush/105-practice/pull/338/files) tab near the top of this page. Once you add those back in, you can add, commit, and push again and then we can get your work merged in.

xoxo,
Sharon

**Comment by opal8000** [comment]
I think it's correct and up to date now. Thanks!

**Comment by acodeinprogress** [comment]
Hey @opal8000,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon

**Pull request merged**